src/components/StyledButtons.js

```diff
@@ -0,0 +1,33 @@
+import styled, {css} from 'styled-components';
+
+const StyledButton = styled.button`
+	${({variant = 'submit'}) =>
+		variant === 'submit' &&
+		css`
+			position: absolute;
+			right: 40px;
+			bottom: 70px;
+			border: none;
+			background-color: transparent;
+			cursor: pointer;
```

Comment:
You are repeating these styles for every button variant. You could simply add this to the `StyledButton` (before the variants) as default styling and apply the more specific styles on top of that according to the chosen variant.